src/store/modules/draft/set/set-stx.js

```diff
@@ -45,15 +45,15 @@ export default {
       appendCard(archive)
     }
 
-    // lesson card, no documentation on the percentages of these cards so using the same as mystical archive
+    // lesson card, percentages created from reverse engineering public sample of 8647 Arena Sealed Pools
     let lesson = undefined;
     const lessonRng = Math.random();
-    if (lessonRng <= 0.67) {
-      lesson = selectCards([commonLesson,uncommonLesson], 1)[0];
-    } else if (lessonRng <= (0.67 + 0.264)) {
-      lesson = selectCards([rareLesson,uncommonLesson,commonLesson], 1)[0];
+    if (lessonRng <= 0.9244) {
+      lesson = selectCards(commonLesson, 1)[0];
+    } else if (lessonRng <= (0.9244 + 0.0684)) {
+      lesson = selectCards(rareLesson, 1)[0];
     } else {
-      lesson = selectCards([mythicLesson,rareLesson,uncommonLesson,commonLesson], 1)[0];
```

Comment:
We actually still need the additional entries in the array (e.g. `[mythicLesson,rareLesson,uncommonLesson,commonLesson]` is correct). This isn't saying choose from among those rarities, it's rather saying pick a mythic lesson, if none available then pick a rare lesson, and so on. This important b/c many people prefer to draft w/o rares and many custom cardpools will have very few mythics. If you have a normal cardpool though then you will always get a mythic in this slot.

Similarly, any time we specify rare lesson we need the other pickers as backups, and anytime we select for an uncommon lesson we need the common picker as a backup.

Reply:
Ah, okay, that makes sense. Since uncommon lessons are theoretically never supposed to appear in the lesson slot, should they be omitted as backups in this specific case?